ajax/libs/alchemyjs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "alchemyjs",
+  "version": "0.4.2",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/graphalchemist/alchemy.git"
```

Comment:
Url is case sensitive, and I would like to use https instead of http, so it should be `"https://github.com/GraphAlchemist/Alchemy.git"`.